src/execution/types.ts

```diff
@@ -129,11 +129,11 @@ export interface IncrementalStreamResult<
 }
 
 export interface FormattedIncrementalStreamResult<
-  TData = Array<unknown>,
+  TData = unknown,
   TExtensions = ObjMap<unknown>,
 > {
   errors?: ReadonlyArray<GraphQLFormattedError>;
-  items: TData;
+  items: ReadonlyArray<TData>;
```

Comment:
Everything else seems to use `ReadonlyArray`, so I figured this should probably be the same instead of `Array` like it was before.